samples/HostConfig/sample.json

```diff
@@ -14,7 +14,7 @@
 	"supportsInteractivity": true,
 	"fontTypes": {
 		"default": {
-			"fontFamily": "'Segoe UI', 'Roboto', 'Oxygen', 'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue', sans-serif",
+			"fontFamily": "'Segoe UI', 'sans-serif', 'Oxygen', 'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue'",
```

Comment:
so are we removing support for `Roboto`?

Reply:
For some undocumented reasons, when trying to set the font as roboto the thing crashes, sans-serif is the consumer friendly name for roboto, hence the change